webapp/channels/src/packages/mattermost-redux/src/utils/user_utils.ts

```diff
@@ -160,7 +160,7 @@ export function nameSuggestionsForUser(user: UserProfile): string[] {
     profileSuggestions.push((user.nickname || '').toLowerCase());
     const positionSuggestions = getSuggestionsSplitBy((user.position || '').toLowerCase(), ' ');
     profileSuggestions.push(...positionSuggestions);
-    const email = (user.email || '').toLowerCase();
+    const email = (user.email || '').toLowerCase().split('@')[0];
```

Comment:
You added the `split('@')` here, but there's another email.split('@') right below. Should we remove that?

Reply:
Good catch. That block is dead code because `email` is guaranteed not to include an at-sign so `split` will always be a one-item array.

Also, digging back as far as I can find, that was actually [intentionally added to match on the email domain for some reason](https://github.com/mattermost/mattermost-redux/pull/70/files#diff-ef3e81f15601316c36c5ba082471baa3ee4026819a9db5dcf2afd681468372b9R94). The previous naming made that much easier to read than the current one though